packages/forms-web-app/src/middleware/check-appeal-exists.js

```diff
@@ -33,7 +33,7 @@ const checkPathAllowed = (req, res, next) => {
 		return next();
 	}
 
-	return res.redirect('/before-you-start');
+	return res.redirect('/');
```

Comment:
if no appeal in session return to home page, should be caught by new login middleware but felt sensible to change here too